src/Module/Movies.php

```diff
@@ -539,7 +731,7 @@ private function _getMovieROI($options) {
             !isset($options['x2']) && !isset($options['y2']) &&
             !isset($options['x0']) && !isset($options['y0']) &&
             !isset($options['width']) && !isset($options['height']) ) {
-            $defaultOptions = $this->_getDefaultROIOptions($this->_params['imageScale']);
+            $defaultOptions = $this->_getDefaultROIOptions($options['imageScale']);
```

Comment:
Double check if this should be params or options. Validator looks like it should be params...

Reply:
never mind, options is an input param, this is ok.